cves/kernel/CVE-2021-3640.yml

```diff
@@ -278,8 +289,9 @@ sandbox:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: |
+    Did not find anything that dealt with access or privileges.
```

Comment:
One of the consequence of this vulnerability is that a user can escalate their privileges. I think this can be a candidate for a sandbox violation.